ResearchKit.xcodeproj/project.pbxproj

```diff
@@ -2251,8 +2280,9 @@
 			isa = PBXHeadersBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				244EFAD21BCEFD83001850D9 /* ORKAnswerFormat_Private.h in Headers */,
```

Comment:
Hm weird, that file is correctly placed into the private headers group within my project. Let's see if things get fixed with my next commit